jobs/JEVS_MESOSCALE_STATS

```diff
@@ -88,11 +88,11 @@ if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
 elif [ $MODELNAME = sref ]; then
   export VDATE=${VDATE:-$PDYm2}
 
-  export COMIN=${COMIN:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT/$RUN
+  export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}/prep/$COMPONENT/$RUN
   export COMINsref=${COMINsref:-$(compath.py  sref/$sref_ver)}
   export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
-  export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
-  export COMINsrefmean=${COMINsrefmean:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT
+  export EVSINccpa=${EVSINccpa:-$(compath.py  ccpa/${ccpa_ver})}
+  export EVSINsrefmean=${EVSINsrefmean:-$(compath.py  $COMIN)}/prep/$COMPONENT
```

Comment:
And similar to the other EVSIN variables, no need for compath.py. This line should be
```
export EVSINsrefmean=${EVSINsrefmean:-$COMIN/prep/$COMPONENT}
```